src/slider.jsx

```diff
@@ -323,17 +314,84 @@ const Slider = React.createClass({
       document.addEventListener('touchcancel', this._dragTouchEndHandler, false);
     }
     this._onDragStart(e);
+
+    // Cancel scroll and context menu
+    e.preventDefault();
   },
 
   _onHandleMouseDown(e) {
     if (document) {
       document.addEventListener('mousemove', this._dragHandler, false);
       document.addEventListener('mouseup', this._dragEndHandler, false);
-      this._toggleSelection('none');
+
+      // Cancel text selection
+      e.preventDefault();
+
+      // Set focus manually since we called preventDefault()
+      ReactDOM.findDOMNode(this.refs.handle).focus();
     }
     this._onDragStart(e);
   },
 
+  _onHandleKeyDown(e) {
+    const pageDown = 34;
```

Comment:
I'm not the biggest fan of the current `utils/key-code.js` utility, but maybe we could reuse that utility for now rather than declaring these constants for now since that utility is being used in other parts of the code base. Either way, these consts could probably be declared at the top of the file rather than in the key down handler.